core-plugins/widgets/FTP-batchsource.json

```diff
@@ -4,7 +4,7 @@
   },
   "configuration-groups": [
     {
-      "label": "FTP Batch Source Properties",
+      "label": "Basic",
       "properties": [
         {
           "widget-type": "textbox",
```

Comment:
can you add a placeholder for all the Reference Names, similar to what was done in https://github.com/caskdata/hydrator-plugins/pull/846